### PR TITLE
Fix flakeheaven plugins error

### PR DESCRIPTION
**Pull request by mcarans**
Attempt to fix https://github.com/flakeheaven/flakeheaven/issues/4

(This time rebased properly)

This is tricky to debug and I have to admit that I don't fully understand the code.

I had added https://github.com/flakeheaven/flakeheaven/blob/master/flakeheaven/_logic/_discover.py#L50-L68 to fix an existing error to do with pylint not being found

However, having looked now, it appears like the pylint plugin is found twice as "flake8-pylint-plugin" and "pylint". I have experimented and found I can remove the extra discovery code, but only if I remove the pylint specific plugin code in flakeheaven.

This PR removes the error with `flakeheaven plugins`. Whether it creates problems with using the pylint plugin I'm not sure and would need someone else to take a look and test.

### CODE REVIEW

**Comment by mcarans** [comment]
@pwoolvett This one is rebased

**Comment by pwoolvett** [comment]
@mcarans 
Just looked at the code. Given your input, I think we got it working at https://github.com/flakeheaven/flakeheaven/pull/10

could you take a look at it?

I haven't validated the dual registration, but I believe its once as an `ast` and once as a `formatter`.

File: flakeheaven/_cli.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def entrypoint(argv: List[str] = None) -> NoReturn:
 
 
 def flake8_entrypoint(argv: List[str] = None) -> NoReturn:
-    """Entrypoint with the same behavior as flake8 (flake8helled)
+    """Entrypoint with the same behavior as flake8 (flake8heavened)
```
Comment (jarojasm95) [inline flakeheaven/_cli.py:47]: I think we'd also need to update `pyproject.toml`:
```
flake8helled = "flakeheaven:flake8_entrypoint"
```
to
```
flake8heavened = "flakeheaven:flake8_entrypoint"
```

**Review by jarojasm95 — COMMENTED** [review]
this makes sense to me, but I'm not too familiar with the `pylint` plugin code - left a minor comment about `flake8heavened`

File: flakeheaven/_cli.py (hunk 1 above)
Comment (mcarans) [reply flakeheaven/_cli.py:47]: Good spot.

**Comment by mcarans** [comment]
Closing this in favour of https://github.com/flakeheaven/flakeheaven/pull/10

**Pull request closed without merging**